### ISSUE TITLE
Single S2 Heart Sound

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9936]
**Preferred term label:**
Single S2 Heart Sound

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C2029938

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0031661

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Tricuspid atresia

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Single second heart sound HP:6000225

**Comment by MickeySegal** [comment]
This is another example of second notices; the original was https://github.com/obophenotype/human-phenotype-ontology/issues/7781#issuecomment-1913167722

**Comment by pnrobinson** [comment]
The problem may have been in the fact that I used a Python script to transfer the Excel file to individual GitHub issues, and I think that for whatever reason there were a few duplicates. Please just ignore in these cases; it is hard to track things like this using GitHub...
